metaflow/plugins/kfp/interruptible_decorator.py

```diff
@@ -1,4 +1,18 @@
+import os
+from typing import Optional
+
+import requests
 from metaflow.decorators import StepDecorator
+from metaflow.metadata.metadata import MetaDatum
+
+
+def _get_ec2_metadata(path: str) -> Optional[str]:
+    response = requests.get(f"http://169.254.169.254/latest/meta-data/{path}")
+    if response.ok:
+        return response.text
+    else:
+        print(f"Error: {response.status_code}")
```

Comment:
Maybe a more verbose error message will help. 

For example `f"Error getting instance metadata {path}: {response.status_code}"`

Reply:
If this error ocurrs, which should be never happen on an EC2 instance, we'd know the path and have more serious AWS issues :) 

also see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

Reply:
Since you coded the error message here I hope it provides more context. If you do not expect this error message to be used, maybe removing this print statement also works fine.

Reply:
thanks for the comment, simplified the code!
if it does fail, we'd get a full stack trace :)